### ISSUE TITLE
Add support for django’s makemessages command with newstyle_gettext

### ISSUE THREAD

**Issue by potpath** [niwinz/django-jinja#131]
With this requirements.txt,
```
Django==1.8.2
Jinja2==2.7.3
django-jinja==1.4.1
```

`manage.py makemessages` can't find translation strings with `newstyle_gettext` format, such as
`{{ _('Hello %(name)s', name=user.name) }}`.

In contrast, `manage.py makemessages` is able to find these translation strings
`{{ _('Hello %(name)s')|format(name=user.name) }}`,
and
`{% trans name=user.name %}Hello {{ name }}{% endtrans %}`.

**Comment by niwinz** [comment]
do you have django_jinja in installed apps ?

**Comment by potpath** [comment]
Yes, this is part of my installed apps
```
INSTALLED_APPS = (
    ...,
    'bootstrap3',
    'django_jinja',
    'django.contrib.admin',
    'django.contrib.auth',
    'django.contrib.contenttypes',
    'django.contrib.sessions',
    'django.contrib.messages',
    'django.contrib.staticfiles',
)
```

**Comment by niwinz** [comment]
Sincerely I don't know that is happens :( Can you provide a little test project for reproduce it?

**Comment by potpath** [comment]
The test project is here:
https://github.com/potpath/django-jinja-makemessages-bug.

From [`django.po`](https://github.com/potpath/django-jinja-makemessages-bug/blob/master/mysite/polls/locale/th/LC_MESSAGES/django.po) and [`test.jinja`](https://github.com/potpath/django-jinja-makemessages-bug/blob/master/mysite/polls/templates/polls/test.jinja),
[`_('newstyle gettext variable %(text)s', text=text)`](https://github.com/potpath/django-jinja-makemessages-bug/blob/master/mysite/polls/templates/polls/test.jinja#L3) is not detected, but
[`_('newstyle gettext static %(text)s', text='foo')`](https://github.com/potpath/django-jinja-makemessages-bug/blob/master/mysite/polls/templates/polls/test.jinja#L4) and others are detected.

So, this bug happens when setting translation variable to another variable instead of static text.

**Comment by niwinz** [comment]
Sorry for very late response. I can't reproduce the bug, the translations are properly detected.

**Comment by potpath** [comment]
With the exact same version in https://github.com/potpath/django-jinja-makemessages-bug/blob/master/requirements.txt ?

Date: Sat, 19 Dec 2015 08:55:02 -0800

**Comment by niwinz** [comment]
Yes, I have downloaded your repository, created new virtualenv with provided requirements and executed makemessages. I have opened the .po file and found the "newstyle gettext variable" there.

**Comment by niwinz** [comment]
Hmm, I have reviset it again now and I'm completly wrong, You are right, it is not detected properly...

**Comment by niwinz** [comment]
Now it should be fixed with 2.1.1

Thanks for report it.

**Comment by potpath** [comment]
Fixed in version 2.1.1, confirmed.
Thank you.
